frontend/src/components/search/search_bar.js

```diff
@@ -41,14 +51,18 @@ class CategorySearch extends React.Component {
         })
         
         return (
-            <div className="ui category search" id="searchbar">
+
+
+           
+            <div className="ui category search" id="searchbar" tabindex="0" onBlur={this.handleBlur} onFocus={this.handleFocus}>
                 <div className="ui icon input">
                     <input className="prompt search" value={this.state.search} onChange={this.updateSearch} type="text" placeholder="Search cards..."></input>
                     
                     <i className="search icon"></i>
                 </div>
-            <div>
-
+                <div className="search-result-cards">
+                {
+                    this.state.open ? (
```

Comment:
Pulled and noticed that this would add an offset to the search bar the div isn't rendered. Fixed if you put the `this.state.open` conditional around the `<div className="ui centered cards" ...` child instead.